Protocol.md

```diff
@@ -279,6 +279,49 @@ The recipient:
 
   These script pub key forms include only standard forms accepted by the wider set of deployed Bitcoin clients in the network, which increase the chances of successful propagation to miners.
 
+## Optional features
+
+Optional features consist of "features" which parties can opt-in to using. These messages are not needed to facilitate construction of a DLCs.
+
+### The `close_dlc` Message
+
+This message contains information about mutual close, which allows the counterparty
+to broadcast a mutual closing transaction.
+
+1. type: ? (`close_dlc_v0`)
```

Comment:
You can use something in the LN experimental range : 32768 - 65535 and mark it as reserved in the waiting room : https://github.com/lightningnetwork/lightning-rfc/issues/716

Reply:
I guess ideally the best would be to use `42784` to follow after the signing message.

Reply:
`42784` seems to be already used by `contract_descriptor_v1`

Reply:
Yes, so until #171 or similar is merged it is taken, but after all the inner structures will use context specific type numbers.

Reply:
what number was ended up being chosen here?

Reply:
At the end I think we decided not to touch message type numbers in #163 so I guess whatever is ok?

Reply:
good point, added to `TODO` section at end of PR